metaci/testresults/robot_importer.py

```diff
@@ -132,7 +134,18 @@ def import_robot_test_results(flowtask, results_dir: str) -> None:
                 testresult.robot_xml = testresult.robot_xml.replace(
                     f'"{screenshot}"', f'"buildflowasset://{asset_id}"'
                 )
+
             testresult.save()
+        results.append(
+            {
+                "name": f"{result['name']}",
+                "group": f"{result['suite']['name']}",
+                "status": f"{result['status'].capitalize()}",
+                "start_time": f"{result['start_time']}",
+                "end_time": f"{result['end_time']}",
+            }
```

Comment:
We need to include result['robot_tags'] as in this data structure. I think we might also want the URL to the individual test result. gus-bus doesn't currently expect it, but it probably should.